cosmos/scheduler/cmd/scheduler.go

```diff
@@ -31,47 +33,61 @@ type Scheduler struct {
 // Run starts the Cosmos Scheduler
 func (s *Scheduler) Run() {
 	log.Println("starting cosmos scheduler")
-	quit := make(chan bool)
 	overlayUpdates, err := s.watchOverlayUpdates()
 	if err != nil {
 		log.Println(err)
 		return
 	}
+	// safely create or update secret on startup
+	go s.configureIngestionSecret(true)
+	go s.checkMongoConnection()
+	go s.watchBucketUpdates()
```

Comment:
correct me if I am wrong, with `watchBucketUpdates` being moved out of the for loop, the  `locationBson` on line 91 doesn't seem to be updating anymore.

Reply:
Yeah you're totally right, I'll fix that. Thank you!